src/command_helpers.rs

```diff
@@ -211,7 +216,10 @@ fn wait_on_child(cmd: &Command, program: &str, child: &mut Child) -> Result<(),
             ));
         }
     };
-    println!("{}", status);
+
+    if cargo_output.warnings {
```

Comment:
`CargoOutput.print_warning` is designed for this use case, it's better to use it instead of if here.

Reply:
oops